.scalafix.conf

```diff
@@ -12,11 +13,23 @@ ignored-files = [
   "FlowPrependSpec.scala",
   "FlowZipSpec.scala",
   "FlowZipWithSpec.scala",
-  "FlowZipWithIndexSpec.scala"
+  "FlowZipWithIndexSpec.scala",
+  "SourceSpec.scala",
+  "StatsSampleSpec.scala",
+  "ActorFlowSpec.scala"
 ]
 
 //ignored packages
 ignored-packages = [
   "doc",
   "jdoc"
 ]
```

Comment:
So these packages will be ignored for the sorting of the imports?

I think we haven't been fully consistent with the naming of those packages. Please also add `docs` and `jdocs`.

Reply:
done，`doc` 、`docs` and `jdoc` are both added.